terraform/aws/load_balancer.tf

```diff
@@ -37,8 +37,8 @@ resource "aws_lb_target_group" "n8n" {
   health_check {
     enabled  = true
     protocol = "HTTP"
-    path     = "/"
-    matcher  = "200-399"
+    path     = "/healthz/readiness"
```

Comment:
Is this supposed to have a z at the end?

Reply:
heh yup, it's what they called it - I think it's a holdover from 1337sp34k from when I was a teenager

Reply:
haha, fair enough! Thanks for confirming - it did look odd.